cmd/cmount/fs.go

```diff
@@ -330,13 +324,15 @@ func (fsys *FS) CreateEx(filePath string, mode uint32, fi *fuse.FileInfo_t) (err
 	if errc != 0 {
 		return errc
 	}
-	file, err := parentDir.Create(leaf, fi.Flags)
+	// translate the fuse flags to os flags
+	osFlags := translateOpenFlags(fi.Flags) | os.O_CREATE
```

Comment:
The previous code was already altering the flags later, i just move it atop so that he same flags get passed to create and open, for consistency.
The flags are compatible, that's why they worked, but having them translated looked the correct way.

Reply:
Closing as part of the old trashed code